query/src/main/resources/application.conf

```diff
@@ -20,12 +20,13 @@ hmda {
 }
 
 db {
-  driver = "slick.driver.H2Driver$"
+  driver = "slick.driver.PostgresDriver$"
 
   db {
-    url = "jdbc:h2:mem:test1;DB_CLOSE_DELAY=-1"
-    driver=org.h2.Driver
-    connectionPool = disabled
-    keepAliveConnection = true
+    # url = "jdbc:postgresql://localhost/hmda?user=postgres&password=postgres"
```

Comment:
Now that the env variable is documented, should the commented URL here still stay?